### ISSUE TITLE
S3 sleep - Check CPU Usage

### ISSUE THREAD

**Issue by newhinton** [bergware/dynamix#82]
I would like to prevent my system going to sleep when i am encoding files. This is entirely local, so i cannot prevent this by a network-check. However, the CPU usage is a good indicator.

Thanks!

**Comment by bjoerns1983** [comment]
Why not simply checking if the array is up and using an array disk as encoding destination?

**Comment by newhinton** [comment]
No, delay only does not work. For some reason status&counter will not prevent sleep. From what i gathered, it should have been written to when it shut down. This might actually be a bug

**Comment by newhinton** [comment]
Yes, there is no cache disk.

Also i am not sure if i am actually writing to a "share". I dont know how unraid works, so i might be directly writing via the mountpoint. (The transcode process is inside a docker container, which writes and reads from a mounted directory and dvd-drives)
